specification/components/r4/schemas/responses/R4-Unauthorized.yaml

```diff
@@ -0,0 +1,18 @@
+description: |
+  Where status code 401 (Unauthorized) is returned then an [NHSDigital-OperationOutcome](https://fhir.nhs.uk/StructureDefinition/NHSDigital-OperationOutcome) will be included in the body, as detailed below.
+  Check diagnostics property for specific information regarding the error.
+  
+  | FHIR issue.details code | FHIR issue.code              | Description                                                                                                 |
+  | ----------------------- | ---------------------------- | ----------------------------------------------------------------------------------------------------------- |
+  | ACCESS_DENIED           | forbidden                    | The request has not been applied because it lacks valid authentication credentials for the target resource. |
```

Comment:
I wonder if the issue code here should be `login` as this would indicate that the client needs to reauthenticate? I think the description should also indicate this "The request could not be authenticated due to either no credentials being provided or the provided credentials no longer being valid. Callers receiving this code should reauthenticate".

Reply:
I agree - changed to suggested issue code and description.